src/wbxml_parser.c

```diff
@@ -2479,7 +2479,8 @@ static WBXMLError decode_opaque_content(WBXMLParser  *parser,
     case WBXML_LANG_SYNCML_SYNCML11: 
     case WBXML_LANG_SYNCML_SYNCML12: 
         /* NextNonce */
-        if ((parser->current_tag->wbxmlCodePage == 0x01) &&
+        if ((parser->current_tag) &&
```

Comment:
@SlavekB thanks for the PR, very nice! :+1:

I tried running the tests without that^^ part of the patch and everything still passes. So it seems that part is not *strictly* related. I would like to vote to make that a separated pull request and to move it out of here for clarity, but just my two cents. Thanks a lot for working on this! :+1:

Reply:
This crash occurred when a damaged wbxml files was generated during tests. Therefore I put it as part of this PR. Because the second commit in this PR fixes problem with damaged wbxml files, the crash does not occur now.

Reply:
I would like to keep as it makes the code more robust.